runtime/default.go

```diff
@@ -204,19 +78,53 @@ func (r *runtime) run() {
 			if service.Running() {
 				continue
 			}
-
 			// TODO: check service error
-			log.Debugf("Starting %s", service.Name)
+			log.Debugf("Runtime starting service %s", service.Name)
 			if err := service.Start(); err != nil {
-				log.Debugf("Runtime error starting %s: %v", service.Name, err)
+				log.Debugf("Runtime error starting service %s: %v", service.Name, err)
+			}
+		case event := <-events:
+			log.Debugf("Runtime received notification event: %v", event)
+			// NOTE: we only handle Update events for now
+			switch event.Type {
+			case Update:
+				// parse returned response to timestamp
+				buildTime, err := time.Parse(time.RFC3339, event.Version)
```

Comment:
Should we be parsing out the version or just compare to the old verson and update if its not the same? This assumes a certain version format which we can't guarantee here.

Reply:
I'm not sure how this would work, actually. Right now we assume *newer* build = a build that was released after we had started the service.

If that's changed we need to figure out some version "math" handling version comparisons. Otherwise simple string equal might lead to downgrades or whatnot.

Reply:
I guess we can revisit later. My thinking was just that versions may not be solely built timestamp formatted like docker.